src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -887,7 +887,8 @@ public with sharing class CRLP_RollupUI_SVC {
             Boolean isRolledUp = referencedFieldSet.contains(dfr);
 
             if ((dfr.isUpdateable() && dfr.isCustom() && isValidType && !isRolledUp && !dfr.name.toLowerCase().contains('__system')
-                    && !dfr.name.contains('Latitude__s') && !dfr.name.contains('Longitude__s'))
+                    && !dfr.name.contains('Latitude__s') && !dfr.name.contains('Longitude__s')
+                    && !fieldIsBanned(objectName,dfr.name))
```

Comment:
If it is preferable here to create a new custom method, maybe we should rename to something like 'targetFieldIsAllowed' and avoid the extra work of inversing it since it's a single-use method. 

...While we're at it, why don't we add the check for contains system, latitude and longitude into this method and get rid of a lot of these extra checks in this massive if block?? :) Otherwise let's rename the method to something more specific since technically all of those are 'banned' as well.